src/Tools/babylon.textureTools.ts

```diff
@@ -60,7 +60,7 @@
 
 		public static GetEnvironmentBRDFTexture(scene: Scene) {
 			if (!scene._environmentBRDFTexture) {
-				var texture = new BABYLON.Texture(this._environmentBRDFBase64Texture, scene, true, false, BABYLON.Texture.BILINEAR_SAMPLINGMODE);
+				var texture = Texture.CreateFromBase64String(this._environmentBRDFBase64Texture, "EnvironmentBRDFTexture", scene, true, false, Texture.BILINEAR_SAMPLINGMODE);

```

Comment:
this._environmentBRDFBase64Texture already starts with "data:" so this won't work right?